src/ontology/edits/oeo-shared.omn

```diff
@@ -2488,6 +2507,9 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1390",
         OEO_00000014
     
     
+Class: OEO_00010386
```

Comment:
Should this be deleted?

Reply:
No, it indicates that this class has to be moved to oeo-shared.